pkg/gcp/disk.py

```diff
@@ -137,34 +138,34 @@ async def start_cloning(project, hostname):
     con = create_db_con()
     try:
         bucket = GcpBucket.objects(project=project).allow_filtering()[0]['bucket']
-        accesskey = GcpBucket.objects(project=project).allow_filtering()[0]['access_key']
+        access_key = GcpBucket.objects(project=project).allow_filtering()[0]['access_key']
         secret_key = GcpBucket.objects(project=project).allow_filtering()[0]['secret_key']
         public_ip = Discover.objects(project=project,host=hostname).allow_filtering()[0]['public_ip']
         user = Project.objects(name=project).allow_filtering()[0]['username']
     except Exception as e:
         print("Error occurred: "+str(e))
     load_dotenv()
+    username = Project.objects(name=project)[0]['username']
     mongodb = os.getenv('BASE_URL')
     current_dir = os.getcwd()
-    if hostname == "all":
-        command = "/usr/local/bin/ansible-playbook -i "+current_dir+"/ansible/"+project+"/hosts "+current_dir+"/ansible/gcp/start_migration.yaml -e \"bucket="+bucket+" access_key="+accesskey+" secret_key="+secret_key+" mongodb="+mongodb+ " project="+project+"\""
-    else:
-        command = "/usr/local/bin/ansible-playbook -i "+current_dir+"/ansible/"+project+"/hosts "+current_dir+"/ansible/gcp/start_migration.yaml -e \"bucket="+bucket+" access_key="+accesskey+" secret_key="+secret_key+" mongodb="+mongodb+ " project="+project+"\" --limit "+public_ip+" --user "+user+" --become-user "+user+" --become-method sudo"
-    print(command)
-    process = await asyncio.create_subprocess_shell(command, stdin = PIPE, stdout = PIPE, stderr = STDOUT)
-    await process.wait()
-    machines = BluePrint.objects(project=project).allow_filtering()
-    machine_count = len(machines)
-    flag = True
-    status_count = 0
-    while flag:
-            for machine in machines:
-                if int(machine['status'])>=25:
-                    status_count = status_count + 1
-            if status_count == machine_count:
-                flag = False
-    con.shutdown()
-    return not flag
+    playbook = "start_migration.yaml"
+    stage = "start clone"
```

Comment:
Use start_clone. But cloning might be better as the log file would appear as cloning _log.txt then